### DISCUSSION TITLE
Is there a way to get GenerateResponse full json when calling sagemaker endpoints

### DISCUSSION THREAD

**Discussion by nth-attempt** [huggingface/text-generation-inference#1029] [Q&A] [answered]
The schema here (https://huggingface.github.io/text-generation-inference/#/Text%20Generation%20Inference/generate) shows the response contains details and generated_text. Currently, the output response from the sagemaker endpoints using TGI only return generated_text. Is there a way to get the full output json in the response to a sagemaker endpoint request?

**Comment by nth-attempt** [comment]
Bumping this!

**Comment by OlivierDehaene** [comment]
@philschmid

**Comment by philschmid** [comment, marked as answer]
You need to provide 
```bash
   "decoder_input_details": true,
    "details": true,
``` 
in the parameters to get the details

**Reply by nth-attempt** [reply]
Thank you!
